packages/devtools/lib/src/memory/memory.dart

```diff
@@ -1,14 +1,15 @@
-// Copyright 2018 The Chromium Authors. All rights reserved.
+// Copyright 2019 The Chromium Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
 import 'dart:async';
 import 'dart:collection';
 import 'dart:math' as math;
 
+import 'package:devtools/src/ui/icons.dart';
```

Comment:
you made the same mistake I've made 50X times. Here and elsewhere search and remove any
package:devtools imports replacing with relative imports.

Reply:
Will do I think it was automatically added by IntelliJ.